spec/rails_helper.rb

```diff
@@ -7,9 +7,9 @@
 require 'rspec/rails'
 # Add additional requires below this line. Rails is not loaded until this point!
 require 'action_cable/testing/rspec'
-require 'database_cleaner'
 require 'selenium/webdriver'
 require 'simplecov'
+require 'factory_bot_rails'
```

Comment:
I like this! do you think it worth reflecting this to the base too?

Reply:
we already have it in the base. It's in `spec/spec_helper.rb`